usl_models/usl_models/flood_ml/model.py

```diff
@@ -1,5 +1,6 @@
 """Flood model definition."""
 
+from dataclasses import replace
```

Comment:
our [style guide sez](https://google.github.io/styleguide/pyguide.html#22-imports) to only import modules rather than individual classes or functions

Reply:
Ack, will fix.

I use `from dataclasses import dataclass` in several places. Is this an exception, or should I change those too?

Reply:
The [only exceptions](https://google.github.io/styleguide/pyguide.html#2241-exemptions) are for the typing & collections modules
but I dunno, maybe we can have an honorary one for dataclasses